components/Layout.js

```diff
@@ -31,14 +46,16 @@ const Nav = styled.nav`
   justify-content: center;
 `;
 
-const NavLink = styled(Link)`
-  color: #ffffff;
+const NavLink = styled(({ isActive, ...props }) => <Link {...props} />)`
```

Comment:
This is a fancy approach!

What you see in the docs from Styled Components is the prefixing of the props passed to a styled component with "$" to filter them. But your solution works just as well 👍

Reply:
Thank you Niklas, this is the solution from gpt.